iqm5q_q0.yml

```diff
@@ -10,6 +10,17 @@ qubits: [0]
 
 topology: []
 
+instruments:
+    lo_readout: # For SG0 (Readout)
+        frequency: 5_500_000_000
+        power: null
```

Comment:
Is it possible to omit `power` instead of putting `null`?

Reply:
Seems possible because the LOs use `None` as default value anyway, thanks.